docs/reference/gc-toolkit/onboarding/index.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 0
+---
+
+# Onboarding
+
+This document is your onboarding document to a fresh Guardian Connector instance.
+
+## Introduction
+
+Welcome to your Guardian Connector instance!
+
+This space is your and yours alone, maintained for you by Conservation Metrics Inc.
+
+Guardian Connector is:
+
+- where you keep your data safe (a Data Warehouse),
+- tools to get your data into it,
+- tools to use your data for your benefit.
+
+Each instance will have an **alias**. Within the Guardian Connector ecosystem, we refer to your instance through this codeword of your choosing. That adds an extra level of opaqueness on how to access a partner's infrastructure. The idea is for you to name your instance, and can be called however you want.
+
+From now on, we will refer to it as *alias*.
+
+In order to access, you will use its web address: https://alias.guardianconnector.net/
+
+The different tools will be available at https://*.alias.guardianconnector.net/ where * can be one of the services currently available:
+
+- Explorer: https://explorer.alias.guardianconnector.net/ 
+- Files: https://files.alias.guardianconnector.net/ 
+- Superset: https://superset.alias.guardianconnector.net/
+- Comapeo: https://comapeo.alias.guardianconnector.net/ 
+- Windmill: https://windmill.alias.guardianconnector.net/ 
+
+## Authentication
+
+All the services are protected so no one can access them without your permission.
+
+The first barrier is your Alias. Please do not share this with anyone outside of your organization, as it is the first door to your infrastructure.
+
+We also have a user access list that we will experience in just a second.
+
+For some services, we might have a predefined user/password combination that I will share with you when we get to them.
+
+## Tools
+
+Guardian Connector is equipped with a suite of tools designed to help communities manage their data and workflows. An extensive walkthrough of these tools is available in our documentation: https://conservationmetrics.github.io/gc-docs/reference/gc-toolkit/ 
+
+### Explorer
+
+It is the service that hosts your Mining alerts, as well as your Comapeo projects. Right now you will only be able to see the mining alerts. Let’s take a look at that.
```

Comment:
I would not lead with "it is the service that hosts your Mining alerts." It is much more general purpose than that, and many communities will not have alerts about mining, or alerts at all.

Same logic applies to CoMapeo. 

Can we rephrase this to be generalizable?